src/widgets/dropdown/lv_dropdown.c

```diff
@@ -347,7 +347,7 @@ void lv_dropdown_set_selected(lv_obj_t * obj, uint32_t sel_opt)
     dropdown->sel_opt_id_orig = dropdown->sel_opt_id;
 
     if(dropdown->list) {
-        position_to_selected(obj);
+        position_to_selected(obj, LV_ANIM_OFF);
```

Comment:
For this one, I feel like the public function `lv_dropdown_set_selected` that this is in could get a `lv_anim_enable_t` parameter. It would make it more consistent with other similar APIs such as

- `lv_bar_set_value`
- `lv_roller_set_selected`
- `lv_slider_set_value`
- `lv_gridnav_set_focused`
- `lv_tabview_set_active`
- `lv_tileview_set_tile`

It would be a "breaking change" but a good one, in my opinion.

Reply:
Thought so aswell but didn't want to introduce any breaking changes to the public api
Should I create a new PR with the breaking change or do it in this one ?

Reply:
Ok, let's start with no breaking change and then there can be a follow-up PR. Thanks.